Packs/CommonPlaybooks/Playbooks/playbook-Unisolate_Endpoint_-_Generic.yml

```diff
@@ -1,283 +1,305 @@
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
     skipunavailable: true
-    quietmode: 0
-  "5":
-    id: "5"
-    taskid: 80b6c790-1aef-4d56-8d48-8b0446ac6e31
-    type: playbook
     task:
+      brand: ''
+      description: This playbook unisolates endpoints according to the hostname/endpoint
+        ID that is provided by the playbook input.
       id: 80b6c790-1aef-4d56-8d48-8b0446ac6e31
-      version: -1
+      iscommand: false
       name: FireEye HX - Unisolate Endpoint
-      description: This playbook unisolates endpoints according to the hostname/endpoint ID
-       that is provided by the playbook input. 
       playbookName: FireEye HX - Unisolate Endpoint
       type: playbook
+      version: -1
+    taskid: 80b6c790-1aef-4d56-8d48-8b0446ac6e31
+    timertriggers: []
+    type: playbook
+    view: "{\n  \"position\": {\n    \"x\": 1340,\n    \"y\": 195\n  }\n}"
+  '6':
+    id: '6'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ''
       iscommand: false
-      brand: ""
+      max: 100
+      wait: 1
     nexttasks:
       '#none#':
-      - "3"
+      - '3'
+    note: false
+    quietmode: 0
     scriptarguments:
-      Endpoint_id:
-        simple: ${inputs.Endpoint_ID}
       Hostname:
         simple: ${inputs.Hostname}
     separatecontext: true
-    loop:
-      iscommand: false
-      exitCondition: ""
-      wait: 1
-      max: 100
-    view: |-
-      {
-        "position": {
-          "x": 1330,
-          "y": 195
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
     skipunavailable: true
-    quietmode: 0
-  "6":
-    id: "6"
-    taskid: d075f33e-0924-4fd0-89c8-6dd42358041d
-    type: playbook
     task:
+      brand: ''
+      description: This playbook unisolates endpoints according to the hostname that
+        is provided by the playbook input.
       id: d075f33e-0924-4fd0-89c8-6dd42358041d
-      version: -1
+      iscommand: false
       name: Unisolate Endpoint - Cybereason
-      description: This playbook unisolates endpoints according to the hostname
-       that is provided by the playbook input.
       playbookName: Unisolate Endpoint - Cybereason
       type: playbook
+      version: -1
+    taskid: d075f33e-0924-4fd0-89c8-6dd42358041d
+    timertriggers: []
+    type: playbook
+    view: "{\n  \"position\": {\n    \"x\": 1770,\n    \"y\": 195\n  }\n}"
+  '7':
+    id: '7'
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ''
       iscommand: false
-      brand: ""
+      max: 100
+      wait: 1
     nexttasks:
       '#none#':
-      - "3"
+      - '3'
+    note: false
+    quietmode: 0
     scriptarguments:
+      Device_id:
+        simple: ${inputs.Endpoint_ID}
       Hostname:
         simple: ${inputs.Hostname}
     separatecontext: true
-    loop:
+    skipunavailable: false
```

Comment:
Make sure that the new playbooks have skip if unavailable set to true
for isolate and unisolate

Reply:
changed on both (unisolate and isolate generic PBs )